packages/docs/docs/how-to-use.mdx

```diff
@@ -48,9 +48,3 @@ To transfer an amount to other address, use the `wallet.transfer` method and pas
 <CodeImport file="../examples/Transfer.tsx" lineStart="31" lineEnd="39" />
 
 <Examples.Transfer />
-
-<CodeImport
```

Comment:
Shouldn't we still include getSelectedAccount in the how to use section of the docs